docs/version-mapping.yaml

```diff
@@ -0,0 +1,19 @@
+# This file lists numaflow versions and their minimum supported SDK versions
+# for documentation purposes. Only versions that have backwards incompatible changes are listed
+
+1.2.0-rc1:
+  - Go:     0.7.0-0
+  - Python: 0.7.0a
+  - Java:   0.7.1-0
+1.2.0-rc2:
+  - Go:     0.7.0-0
+  - Python: 0.7.0a
+  - Java:   0.7.1-0
+1.2.0-rc3:
+  - Go:     0.7.0-0
+  - Python: 0.7.0a
+  - Java:   0.7.1-0
+1.2.0-rc4:
+  - Go:     0.7.0-rc2
+  - Python: 0.7.0a1
+  - Java:   0.7.2-0
```

Comment:
Let's rearrange the order so that the latest version is at the top so that it is more relatable to the user.